### PR TITLE
One Single Symbolica

### PR DESCRIPTION
**Pull request by ValentinHirschi**
GAMMALOOP_USER licensing and centralised internal symbolica dependency from our fork installed through `./bin/build_dependencies.sh`.
See new readme for details.

### CODE REVIEW

File: python/gammaloop/base_objects/model.py [hunk 1]
```diff
@@ -359,14 +359,20 @@ def is_ghost(self) -> bool:
     def is_massive(self) -> bool:
         return self.mass.value is None or abs(self.mass.value) > 0.
 
+    @staticmethod
+    def santize_texname(texname: str) -> str:
```
Comment (GraDje13) [inline python/gammaloop/base_objects/model.py:363]: I think there is a typo here, assuming that sanitize_texname was the intended function name :)

**Review by GraDje13 — APPROVED** [review]
Looks good! Only thing I could find after a quick look is the typo :).

File: python/gammaloop/base_objects/model.py (hunk 1 above)
Comment (ValentinHirschi) [reply python/gammaloop/base_objects/model.py:363]: Yeap, good catch!

File: README.md [hunk 2]
```diff
@@ -12,18 +12,18 @@ See the [wiki](https://wiki.alphaloop.ch/) for more information on the project.
 ### > Installation using `pip`
 ```
 pip install gammaloop
+gammaloop --build_dependencies
+source `gammaloop -show_venv`
```
Comment (lcnbr) [inline README.md:16]: Very minor nitpick: isn't show_venv a bit too wordy for a short flag?

File: python/gammaloop/__init__.py [hunk 3]
```diff
@@ -1,10 +1,86 @@
...
+    if not os.path.isfile(os.path.join(gammaloop_root_path, 'dependencies', 'INSTALLED')):
+        print("\nGammaloop dependencies are %snot installed%s. Run '%sgammaloop --build_dependencies%s' to install them first. Exiting.\n" % (
+            CLIColour.RED, CLIColour.END, CLIColour.GREEN, CLIColour.END))
+        sys.exit(1)
+
+    venv_path = os.path.abspath(os.path.join(
+        gammaloop_root_path, 'dependencies', 'venv'))
+
+    if venv_path != os.path.abspath(os.path.join(os.path.dirname(sys.executable), os.path.pardir)):
+        print("%sWARNING:%s It is recommended to run gammaloop within its Python virtual environment, by issueing the command:\n\n%ssource %s/bin/activate%s\n" % (
```
Comment (lcnbr) [inline python/gammaloop/__init__.py:61]: issueing is a misspelling: it should be issuing

File: python/gammaloop/base_objects/model.py (hunk 1 above)
Comment (ValentinHirschi) [reply python/gammaloop/base_objects/model.py]: @GraDje13 Could you try a build from scratch on your system and it worked?

**Review by lcnbr — COMMENTED** [review]
Same for me! Everything looks good. Another small typo notwithstanding.

File: python/gammaloop/__init__.py (hunk 3 above)
Comment (ValentinHirschi) [reply python/gammaloop/__init__.py:61]: I actually saw that but since it was not picked up on by my spellchecker I didn't look it up. Should have trusted myself more ^^

File: README.md (hunk 2 above)
Comment (ValentinHirschi) [reply README.md:16]: right indeed, there is anyway a long verbose option with double dash. Will shorten it to `-venv`
